src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigureTwo implements AreaOfFigure,InfoFigure {
+    //квадрат
+
+    public Square(String color, double firsSide) {
```

Comment:
There is a typo in the parameter name `firsSide`. It should be `firstSide` to maintain clarity and consistency in naming conventions.